### ISSUE TITLE
Picking priority

### ISSUE THREAD

**Issue by emackey** [CesiumGS/cesium#1592]
It would be a useful addition to CZML for objects to have a "picking priority" of some sort.  @mramato and I discussed possibly an enum (High, Default, Low, None) or a number (where 1 is default, and 0 means don't pick).  The viewerDynamicObjectMixin would drill-pick and take the highest-priority pickable object as the picked object.

This feature would be useful for scenes that have a lot of sensor cones, or a single very large sensor cone, for example.  The cone(s) can contain multiple objects, and the user may be more interested in picking the interior objects than the cone itself.  The cone would get "low" priority (but not un-pickable) and would only get picked if there were no other objects under the mouse.

**Comment by ivanprice** [comment]
yep us too.. we're making a measuring tool and it doesn't make any sense that the vertex markers or line are clickable.

**Comment by hpinkos** [comment]
This is something that gets requested in the forum frequently.

https://groups.google.com/forum/?hl=en#!topic/cesium-dev/0XA8qWPQfRE
https://groups.google.com/d/msg/cesium-dev/agtOrHYEhUo/lGlFb5q4EQAJ

**Comment by pierotofy** [comment]
+1

**Comment by dwhipps** [comment]
I'm going to work on this right now. Should have a PR by tomorrow.

**Comment by hpinkos** [comment]
Thanks @dwhipps!  This has been a frequently requested feature, I think a lot of people will be happy to see this added

**Comment by rahwang** [comment]
This came up again on the forum: https://groups.google.com/forum/?hl=en#!topic/cesium-dev/wO_LHnTsmJs

**Comment by dwhipps** [comment]
My PR is still open here: https://github.com/AnalyticalGraphicsInc/cesium/pull/4410

**Comment by svenvarennes** [comment]
Any news on this?

**Comment by pjcozzi** [comment]
@svenvarennes no update on this beyond what is in #4410.  Feel free to test, review, and chime in on that PR.

**Comment by hpinkos** [comment]
Also requested by @githubhanjunjun in #7623

**Comment by OmarShehata** [comment]
Requested again here: https://groups.google.com/d/msg/cesium-dev/CzEQFO5a2cU/M9cEmPG9EgAJ

**Comment by OmarShehata** [comment]
Requested again here: https://groups.google.com/d/msg/cesium-dev/NojRg2rnfUg/JwMkEqkQAwAJ

> I have a geojson file which i load as a baselayer using viewer.datasource  and have several other geojsons that load on top of the baselayer. I want to make the baselayer geojson file not be selected when clicking on the polygon and being shown in the cesium info box. Is there a way to do so.

**Comment by DallasHoff** [comment]
+1

**Comment by ggetz** [comment]
Also requested in https://github.com/CesiumGS/cesium/issues/11022.
